pymeasure/instruments/keithley/keithley2260B.py

```diff
@@ -0,0 +1,71 @@
+from pymeasure.instruments import Instrument
+from pymeasure.instruments.validators import (
+    # truncated_range,
+    # truncated_discrete_set,
+    strict_discrete_set,
+)
+# from pymeasure.adapters import VISAAdapter
+from .buffer import KeithleyBuffer
+
+import logging
+
+log = logging.getLogger(__name__)
+log.addHandler(logging.NullHandler())
+
+
+class Keithley2260B(Instrument):
+    """ Represents the Keithley 2260B Power Supply (minimal implementation)
+    and provides a high-level interface for interacting with the instrument.
+
+    For a connection through tcpip, the socket is hardcoded to 2268,
+        example connection string: 'TCPIP::xxx.xxx.xxx.xxx::2268::SOCKET'
+        the read termination is \n
+
+    .. code-block:: python
+
+        source = Keithley2260B("GPIB::1")
+        source.voltage = 1
+        print(source.voltage)
+        print(source.current)
+        print(source.power)
+        print(source.applied)
+
+    """
+
+    enabled = Instrument.control(
+        "OUTPut?", "OUTPut %d",
+        """A boolean property that controls whether the source is enabled, takes
+        values True or False (1 or 0). """,
```

Comment:
This should only take True/False, not 1/0 as indicated in the docstring.

Reply:
I am not quite sure of the behavior of the validator with the `strict_discrete_set`, the given values and the `map_values` option. 
With the code as it is now, it accepts both `True`/`False` and `1`/`0` - i.e. with the version of pymeasure this was based on. possibly 1 and 0 are directly being interpreted as True/False? or was this changed recently?

Reply:
I thought that giving e.g. `1` would fail when it looks up the input in the defined values map. It's not a big deal.